### ISSUE TITLE
[Accessibility Issue] Button Core Component : When Link is not configured, the button type is missing.

### ISSUE THREAD

**Issue by psm-aig** [adobe/aem-core-wcm-components#1166]
## Bug Report

**Current Behavior**
Core component button - when the link is not configured in the dialog, button type is missing which is leading to accessibility issue.

**Expected behavior/code**
When link is not configured - we should have button type element.

**Environment**
AEM 6.5.2
Core components - 2.10.0
JDK 8

**Possible Solution**
In the below html, we need to have check to include button type if link is not configured
content/src/content/jcr_root/apps/core/wcm/components/button/v1/button/button.html

**Comment by msagolj** [comment]
PR got merged, closing.
